tests/js/common/aql/aql-search-optimize-top-k.js

```diff
@@ -65,42 +71,213 @@ function testOptimizeTopK() {
 
       // testWithoutScore
       for (const v of names) {
-        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 10 RETURN d";
-        print(query);
+        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 100 RETURN d";
         let res = db._query(query).toArray();
-        assertEqual(res.length, 10);
+        assertEqual(res.length, 100);
+      }
+
+      for (const v of names) {
+        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 1500 RETURN d";
+        let res = db._query(query).toArray();
+        assertEqual(res.length, 1500);
       }
-    },
 
-    tearDownAll: function () {
       for (const v of names) {
-        db._dropView(v);
+        const query = "FOR d IN " + v + " SEARCH d.f == 'a' OPTIONS {waitForSync:true} LIMIT 1500, 1000 RETURN d";
+        let res = db._query(query).toArray();
+        assertEqual(res.length, 1000);
       }
-      db._drop("c");
+    },
+
+    tearDownAll: function () {
+      db._useDatabase('_system');
+      try { db._dropDatabase(dbName); } catch (err) { }
     },
 
     testWithTFIDF: function () {
       for (const v of names) {
-        const query = "FOR d IN " + v + " SEARCH d.f == 'a' SORT TFIDF(d) LIMIT 10 RETURN d";
-        print(query);
+        const query = "FOR d IN " + v + " SEARCH d.f == 'a' SORT TFIDF(d) LIMIT 100 RETURN d";
```

Comment:
WAND would not be used here as default sorting is ASC. I guess DESC was expected here?